### DISCUSSION TITLE
Alternate field name for 'tags'

### DISCUSSION THREAD

**Discussion by jmatthewpryor** [estruyf/vscode-front-matter#260] [Q&A] [answered]
I am trying to have a tags field called tag rather than tags

Despite creating new default content type, the Markdown file always ends up writing a field called 'tags'

```
  "frontMatter.taxonomy.contentTypes": [
    {
      "name": "default",
      "previewPath": null,
      "pageBundle": false,
      "fields": [

[[SNIP]]

        {
          "title": "My Tags",
          "name": "tag",
          "type": "tags"
        },
```

Should it be possible to use an alternate field name? Or have more than on tags type field?

**Comment by estruyf** [comment, marked as answer]
Thanks @jmatthewpryor for opening this discussion. It seems I overlooked this and need to be fixed. Created an issue for it to track the status: https://github.com/estruyf/vscode-front-matter/issues/261
